scripts/lib.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Module containing code shared across various shell scripts
+# Execute functions from this module via the script do.sh
+
+# Check if a command is available
+require() {
+    command -v "$1" >/dev/null 2>&1
+}
+
+configure-sentry-cli() {
```

Comment:
Is this function necessary? I know that it's called by the pyenv script but does it need to be in Snuba?

Reply:
The pyenv setup script requires it

https://github.com/getsentry/snuba/pull/2012/files#diff-412b37f70434d5f5ec589787dda24887d69ec454a6d53045a9459edf0478089bR112

I could try and see if it'll work without it but I assumed that it was in there for a reason (so I kept it)

Reply:
Yeah I saw that it's in the pyenv script, I'm just not sure why it's necessary. `sentry-cli` isn't called anywhere in the script.

Reply:
I think this is used here to send errors to sentry during the install script
https://docs.sentry.io/product/cli/send-event/#bash-hook
Adding a dependency on a sentry tool on the snuba code base for that (considering we only set up the python env) for that seem a bit overkill but I won't block it for that reason

Reply:
Oh I see. Yeah OK well that makes sense.